envs/monkey_zoo/blackbox/test_configurations/depth_1_a.py

```diff
@@ -30,7 +30,11 @@
 
 def _add_exploiters(agent_configuration: AgentConfiguration) -> AgentConfiguration:
     exploiters: Dict[str, Mapping] = {
-        "WMI": {"agent_binary_upload_timeout": 30, "smb_connect_timeout": 5},
+        "WMI": {
+            "agent_binary_upload_timeout": 30,
+            "smb_connect_timeout": 5,
+            "wmi_connect_timeout": 10,
```

Comment:
Not sure if we want another option for such a low-level setting

Reply:
I wasn't sure as well. We can leave one and use it in two places. I don't mind that

Reply:
I don't think it needs to be configurable at all based on where it's being used.

Reply:
What should be the hard-coded value for it? The default for smb is 15 seconds